Wiki/Modules/FvwmBacker/index.md

```diff
@@ -3,14 +3,94 @@ title: FvwmBacker
 type: module
 weight: 430
 description: |
-  This module will change the backgound/wallpaper when changing desktops.
+  This module will change the background/wallpaper when changing desktops.
   Any command can be executed when changing desktops so this module can
   be used for more than just switching the background.
 ---
 # FvwmBacker
 
-The FvwmBacker module provides functionality to change the  background when
-changing  desktops.  Any command can be executed to change the backgrounds.
-Actually, any arbitrary command can be sent to fvwm to execute, so  you
+* TOC
+{:toc}
+
+The FvwmBacker module provides functionality to change the background when
+changing  desktops. Any command can be executed to change the backgrounds.
+Actually, any arbitrary command can be sent to fvwm to execute, so you
 could also do things such as changing window border colors, etc.
 
+You can use the FvwmPager to move from screen to screen through each virtual
+desktop set. Now that you can move easily between the desktops you can further
+personalize them by giving them individual background colors.
+
+{% fvwm2rc %}
+*FvwmBacker: Command (Desk 2) -solid white
+{% endfvwm2rc %}
+
+The changes will not be implemented until you instruct Fvwm to update the screens
+during the initialization:
+
+{% fvwm2rc %}
+AddToFunc InitFunction I Module FvwmBacker
+{% endfvwm2rc %}
```

Comment:
You don't want to run modules from `InitFunction`, they need to be launched each time. Add this to `StartFunction`. In general the use of `InitFunction` is discouraged, instead use `Test (Init) Command` in the `StartFunction`, as done in the default-config.